### PR TITLE
Add GeoMesa Kafka Streams quickstart

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-tutorials-kafka/geomesa-tutorials-kafka-quickstart/pom.xml
Comment (aheyne) [inline geomesa-tutorials-kafka/geomesa-tutorials-kafka-quickstart/pom.xml]: Revert

File: pom.xml
Comment (aheyne) [inline pom.xml:26]: @elahrvivaz I guess we want to wait for a GM release before we merge this?

Comment (elahrvivaz) [reply pom.xml:26]: we can bump this to 3.5.0-SNAPSHOT and it will pull down the nightly build

Comment (elahrvivaz) [reply pom.xml:26]: and we'll bump it to 3.5.0 when we release

File: geomesa-tutorials-kafka/geomesa-tutorials-kafka-quickstart/pom.xml
Comment (elahrvivaz) [inline geomesa-tutorials-kafka/geomesa-tutorials-kafka-quickstart/pom.xml]:
```suggestion
                                <argument>org.geomesa.example.kafka.KafkaQuickStart</argument>
```

File: geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/src/main/java/org/geomesa/example/kafka/KafkaStreamsQuickStart.java
Comment (elahrvivaz) [inline geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/src/main/java/org/geomesa/example/kafka/KafkaStreamsQuickStart.java]:
```suggestion
    private Properties streamsConfig() {
        Properties properties = new Properties();

        properties.put(StreamsConfig.APPLICATION_ID_CONFIG, "test");
        properties.put(StreamsConfig.CLIENT_ID_CONFIG, "test-client");
        properties.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, params.get("kafka.brokers"));
```

File: geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/README.md
Comment (jnh5y) [inline geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/README.md:4]: Is there an associated PR for the docs change with this PR?

Comment (aheyne) [reply geomesa-tutorials-kafka/geomesa-tutorials-kafka-streams-quickstart/README.md:4]: https://github.com/locationtech/geomesa/pull/2903

**Review by elahrvivaz — APPROVED** [review]

**Pull request merged**
